File: python/fizzbuzz.py
for i in range(1,101):
  pad = ""
  if len(str(i)) == 1:
    pad = "  "
  elif len(str(i)) == 2:
    pad = " "
  if i % 15 == 0:
    print(f'{pad}{i} - fizzbuzz')
  elif i % 3 == 0:
    print(f'{pad}{i} - fizz')
  elif i % 5 == 0:
    print(f'{pad}{i} -     buzz')
  else:
    print(f'{pad}{i}')

File: perl/fizzbuzz.pl
#!/usr/bin/perl

for my $i (1..100) {
  if ($i % 15 == 0) {print "$i fizzbuzz\n"}
  elsif ($i % 3 == 0) {print "$i fizz\n"}
  elsif ($i % 5 == 0) {print "$i     buzz\n"}
  else {print "$i \n"}
}